api/locations.md

```diff
@@ -282,15 +312,23 @@ Status: 200 OK
 
 ### Parameters
 
-Parameter | Description
---- | ---
-`sw` | Comma separated latitude and longitude of the southwest position. _Required. Will become optional._
-`ne` | Comma separated latitude and longitude of the northeast position. _Required. Will become optional._
-`date_start` | Filter parking spaces for the start date. _Optional._
-`date_end` | Filter parking spaces for the end date. _Optional._
+Parameter      | Value      | Description
+-------------- | ---------- | ----
+`sw`           | `decimal`  |Comma separated latitude and longitude of the southwest position. _Required.
+`ne`           | `decimal`  |Comma separated latitude and longitude of the northeast position. _Required.
+`date_start`   | `datetime` | Filter parking spaces for the start date. _Optional._
+`date_end`     | `datetime` | Filter parking spaces for the end date. _Optional._
```

Comment:
From a form perspective both are optional, if you provide only date_start without date_end it should work as if date_start was not provided. added a comment for that